client/src/Event/CreateEvent.tsx

```diff
@@ -227,11 +186,15 @@ export const CreateEvent: React.FC = () => {
                 >
                     <LabeledInput
                         label={"event name"}
-                        error={errors.eventName && "event name is required"}
+                        error={
+                            errors.eventName &&
+                            "event name is required and must be less than 255 characters"
+                        }
                         inputProps={{
                             placeholder: "Enter Event Name...",
                             ...register("eventName", {
                                 required: true,
+                                maxLength: 255,
```

Comment:
Can this also use MAX_NAME_LENGTH? Maybe a more generic MAX_TEXT_BOX_LENGTH or something which can just be 255?

Reply:
62034b3